### PR TITLE
Fix some Bash warnings in Win installer

### PR DESCRIPTION
**Pull request by declension**
Thanks to shellcheck really...

### CODE REVIEW

File: dev-utils/win_installer/_base.sh [hunk 1]
```diff
@@ -203,7 +206,7 @@ function cleanup_after {
     rm -Rf "${MINGW_ROOT}"/libexec
     rm -Rf "${MINGW_ROOT}"/share/gtk-doc
     rm -Rf "${MINGW_ROOT}"/include
-    rm -Rf "${MINGW_ROOT}"/var
+    rm -Rf "${MINGW_ROOT:?}"/var
```
Comment (lazka) [inline dev-utils/win_installer/_base.sh:209]: Do you know why it does the error checking only in this line and not the 50 others?

File: dev-utils/win_installer/_base.sh [hunk 2]
```diff
@@ -111,7 +112,7 @@ flake8==3.9.2
 "
 
     build_pip install --no-binary ":all:" \
-        --force-reinstall $(echo "$PIP_REQUIREMENTS" | tr ["\\n"] [" "])
+        --force-reinstall "$(echo "$PIP_REQUIREMENTS" | tr "\\n" " ")"
```
Comment (lazka) [inline dev-utils/win_installer/_base.sh:115]: that's not the same, the lines need to be passed as separate arguments

File: dev-utils/win_installer/_base.sh (hunk 1 above)
Comment (declension) [reply dev-utils/win_installer/_base.sh:209]: Yes it "knows" about certain top-level directories that would be a disaster to remove and `/var` is one of them. I guess we could add this everywhere. 
Though might be easier to `test` for it earlier and exit if empty?

File: dev-utils/win_installer/_base.sh (hunk 2 above)
Comment (declension) [reply dev-utils/win_installer/_base.sh:115]: Oh do you mean the outer quotes?

File: dev-utils/win_installer/_base.sh (hunk 1 above)
Comment (lazka) [reply dev-utils/win_installer/_base.sh:209]: I see, not sure..

File: dev-utils/win_installer/_base.sh (hunk 2 above)
Comment (lazka) [reply dev-utils/win_installer/_base.sh:115]: Yes

Comment (declension) [reply dev-utils/win_installer/_base.sh]: Fixed, thanks!

**Comment by lazka** [comment]
lgtm

**Pull request merged**